ggml-cuda.cu

```diff
@@ -11604,3 +11606,17 @@ GGML_CALL int ggml_backend_cuda_reg_devices() {
     }
     return device_count;
 }
+
+extern "C" GGML_CALL void ggml_free_cublas(void);
+GGML_CALL void ggml_free_cublas(void) {
+    for (int id = 0; id < g_device_count; ++id) {
+#if !defined(GGML_USE_HIPBLAS)
```

Comment:
I think this should be

```suggestion
#if !(defined(GGML_USE_HIPBLAS) && defined(__HIP_PLATFORM_AMD__))
```

but then the condition for vmm in `ggml-cuda.cu` should also be changed.

Reply:
done.